android/app/src/main/java/app/covidshield/module/ExposureNotificationModule.kt

```diff
@@ -195,6 +199,24 @@ class ExposureNotificationModule(context: ReactApplicationContext) : ReactContex
         }
     }
 
+    @ReactMethod
+    fun provideDiagnosisKeys(diagnosisKeysURLs: ReadableArray, promise: Promise) {
+        promise.launch(this) {
+            try{
+                val files = diagnosisKeysURLs.parse(String::class.java).map { File(it) }
+                val diagnosisKeyFileProvider = DiagnosisKeyFileProvider(files)
+                exposureNotificationClient.provideDiagnosisKeys(diagnosisKeyFileProvider)
```

Comment:
tested locally and `provideDiagnosisKeys` never seemed to return to react native. I added:
```
promise.resolve(null)
```
here and that seemed to fix it.